src/main/java/com/hedera/hashgraph/sdk/FunctionResult.java

```diff
@@ -73,6 +73,12 @@ public ByteString getBytes(int valIndex) {
         return getByteString(offset + 32, offset + 32 + len);
     }
 
+    /** Get the nth returned value as a string  from byte 32*/
+    public String getBytes32(int valIndex) {
```

Comment:
`String` isn't the right return type for `bytes32` though. This should return `ByteString` which the user can then call `toStringUtf8` on themselves if they like.